docs/user-guide/02-conventional-features.md

```diff
@@ -620,10 +620,13 @@ Like with the previous `vertexMain` example, mixed entry point setups also suppo
 
 However, because of certain systematic differences between entry point types, a uniform being _global_ or _local_ will have very important consequences on the underlying layout and behavior.
 
-For most all entry point types, D3D12 and Vulkan use one common root signature to define both global and local uniform parameters. For these cases, Slang maps uniforms to this common root signature. 
+For most all entry point types, D3D12 will use one common root signature to define both global and local uniform parameters. 
+Likewise, Vulkan descriptors will bind to a common pipeline layout. For both of these cases, Slang maps uniforms to the common root signature / pipeline layout. 
 
-However, for ray tracing entry points, these parameters map to either _global_ root signatures or to _local_ root signatures, with the latter being stored in the shader binding table.
-When entry points match a "ray tracing" type, we bind uniforms which are in the _global_ scope to the _global_ root signature, while uniforms which are _local_ are bound to the _local_ root signature.
+However, for ray tracing entry points and D3D12, these parameters map to either _global_ root signatures or to _local_ root signatures, with the latter being stored in the shader binding table.
+In Vulkan, D3D12's global root signatures translate to a shared ray tracing pipeline layout, while local root signatures map again to shader binding table records. 
+
+When entry points match a "ray tracing" type, we bind uniforms which are in the _global_ scope to the _global_ root signature (or ray tracing pipeline layout), while uniforms which are _local_ are bound to shader binding table records, which depend on the underlying runtime record indexing. 
```

Comment:
When compiling a ray tracing entry-point, Slang allocates uniforms defined in the global scope from the global root signature (or ray tracing pipeline layout), and allocates uniform entry-point parameters from local root signature or shader binding table.

Reply:
>  local root signature or shader binding table.

There is no distinction between the local root signature and the shader binding table. D3D12 uses both interchangeably, while Vulkan just uses "shader binding table".